docs/index.md

```diff
@@ -13,13 +13,94 @@ myst:
 
 This is the community-maintained documentation for the Plone content management system.
 
-This documentation is for:
+:::::{grid} 1 1 2 2
+:gutter: 3 3 4 5
 
-- Editors who write and update content on a site
-- Site administrators who install Plone and add-ons, and set up a site
-- Designers who create site themes
-- Deployers who configure servers for site hosting
-- Developers who customize a site’s capabilities, create add-ons, and contribute to Plone itself
+::::{grid-item-card} 
+:link: https://6.docs.plone.org/install/index.html
+{octicon}`zap;1.5em;sd-text-info`
+Get started
+^^^
+This part of the documentation helps you find the best way to get started with Plone, depending on what you want to do.
++++
+```{button-link} https://6.docs.plone.org/install/index.html
+:color: primary
+:expand:
+Go to Get started
+```
+::::
+
+::::{grid-item-card} 
+:link: http://127.0.0.1:8050/user-guide/index.html
+{octicon}`person;1.5em;sd-text-info`
+User guide 
+^^^
+This part of the documentation helps you understand how to use Plone effectively, covering essential features, navigation, content management, and common tasks for users and administrators.
++++
+```{button-link} http://127.0.0.1:8050/user-guide/index.html
+:color: primary
+:expand:
+Go to User guide
+```
+::::
+
+::::{grid-item-card}
+:link: https://6.docs.plone.org/developer-guide/index.html
+{octicon}`person;1.5em;sd-text-info`
+Developer guide
+^^^
+This part of the documentation provides information for how to develop in Plone.
++++
+```{button-link} https://6.docs.plone.org/developer-guide/index.html
+:color: primary
+:expand:
+Go to Developer guide
+```
+::::
+
+::::{grid-item-card} 
+:link: http://127.0.0.1:8050/reference-guide/index.html
+{octicon}`bookmark;1.5em;sd-text-info`
+Reference guide 
+^^^
+This part of the documentation provides explanation of APIs, functions, modules, and objects included in Plone, describing what they are and what they do.
++++
+```{button-link} http://127.0.0.1:8050/reference-guide/index.html
+:color: primary
+:expand:
+Go to Reference guide
+```
+::::
+
+::::{grid-item-card} 
+:link: https://6.docs.plone.org/conceptual-guides/index.html
+{octicon}`light-bulb;1.5em;sd-text-info`
+Conceptual guides 
```

Comment:
I would put Conceptual guides before Developer guide. It's probably a good idea to understand some concepts before you dive into developing.

Reply:
For some conceptual guides, like choosing a user interface or compare buildout and pip, I'd agree. In this case, a how-to guide includes a link to the relevant conceptual guide for deeper understanding if the reader wants to go there. Others require advanced technical knowledge to understand, or describe how sausage is made, and for those I'd disagree.

When I develop, I seek how-tos first, API references second, source code third, and as a last resort conceptual guides. I think that's what most developers do, try stuff out, and when it doesn't work, dig deeper and deeper.

Reply:
@stevepiercy You may be right and I don't feel strongly about this. I'll resolve the comment, and we can always revisit it later.